shared/actions/platform-specific.native.js

```diff
@@ -97,8 +115,31 @@ function displayNewMessageNotification(text: string, convID: ?string, badgeCount
   })
 }
 
+let lastPush = null
 function configurePush() {
   return eventChannel(dispatch => {
+    if (RNEmitter) {
+      // If android launched due to push
+      RNEmitter.addListener('androidIntentNotification', payload => {
```

Comment:
payload parameter not used?

Reply:
oh yeah, lemme move that check into the java side